### PR TITLE
OR to SDS ADF Template

### PR DESCRIPTION
**Pull request by bramchandram**
Created OR to SDS ADF template. The file name is 'OR_to_SDS_ADF_Template.json'.
The file is available under the OR folder. Please let me know if you face any issues finding the file.
And I was deployed the template, you can check that deployed resources in "ramtest" resource group.

And please review the script and let me know the required changes.

### CODE REVIEW

**Comment by bramchandram** [comment]
> @bramchandram please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

**Review by AyronJohnson — CHANGES_REQUESTED** [review]
Please fix first round of review

File: CustomSolutions/SDS_ADF_ETL_Integrations/Canvas/README.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# Solution Guide: Canvas API Provisiong Reports Data extract and Upload to SDS
```
Comment (AyronJohnson) [inline CustomSolutions/SDS_ADF_ETL_Integrations/Canvas/README.md:1]: This is the instructions for Canvas.  Please make the changes from the previous OneRoster document.

File: CustomSolutions/SDS_ADF_ETL_Integrations/OR/OR_to_SDS_ADF_Template.Json [hunk 2]
```diff
@@ -10,6 +10,10 @@
             "defaultValue": "storsds",
             "type": "String"
         },
+        "IMSOneRosterValidator_properties_typeProperties_url": {
+            "type": "string",
+            "defaultValue": "https://partnertest.infinitecampus.org/campus/oneroster/microsofttest/ims/oneroster/v1p1"
```
Comment (AyronJohnson) [inline CustomSolutions/SDS_ADF_ETL_Integrations/OR/OR_to_SDS_ADF_Template.Json:15]: A url should not be present here.

File: CustomSolutions/SDS_ADF_ETL_Integrations/OR/OR_to_SDS_ADF_Template.Json [hunk 3]
```diff
@@ -7041,7 +6994,7 @@
                                     "userProperties": [],
                                     "typeProperties": {
                                         "pipeline": {
-                                            "referenceName": "Transform_to_Optionals_SDS_Vnext_csv",
+                                            "referenceName": "Transform_to_Optionals_SDS_csv",
```
Comment (AyronJohnson) [inline CustomSolutions/SDS_ADF_ETL_Integrations/OR/OR_to_SDS_ADF_Template.Json:6993]: NIT: Remove "s" from all instances of "Optionals"

File: CustomSolutions/SDS_ADF_ETL_Integrations/Canvas/README.md (hunk 1 above)
Comment (AyronJohnson) [inline CustomSolutions/SDS_ADF_ETL_Integrations/Canvas/README.md:1]: This is the instructions for Canvas. Please remove and convert the pdf from the previous OneRoster document to markdown.

**Review by AyronJohnson — CHANGES_REQUESTED** [review]

**Comment by AyronJohnson** [comment]
@bramchandram please see review comments.  Also, I saw the corrections below are needed in addition.  Thanks

- line 313 AppName not showing because it is a tag.  Need to escape
- lines 125, 128 replace SMS with LMS?
- line 130 remove line break tag
- line 136 replace sentence with  "Once the ADF and storage accounts are created under stonerostersds, create a container named as refcodes to import refCodes.csv used for subject in classes."

File: CustomSolutions/SDS_ADF_ETL_Integrations/OR/OneRoster_README.md [hunk 4]
```diff
@@ -110,10 +109,10 @@ following steps:
...
-| entraAppClientId                 | string   | The clientId for the app registration. Found at Azure Active Directory -> App registrations -> <AppName>-> Overview -> Application (client) ID                                                                                                                                                                                                                                                                  |
+| entraAppClientId                 | string   | The clientId for the app registration. Found at Azure Active Directory -> App registrations <br/>-> \<AppName>\ -> Overview -> Application (client) ID                                                                                                                                                                                                                                                                  |
```
Comment (AyronJohnson) [inline CustomSolutions/SDS_ADF_ETL_Integrations/OR/OneRoster_README.md:112]: The backslash \ is appearing after the >.  Please move.

File: CustomSolutions/SDS_ADF_ETL_Integrations/OR/OneRoster_README.md [hunk 5]
```diff
@@ -122,18 +121,18 @@ following steps:
...
 | expirationDateTime               | String   | The date when SDS should stop syncing data based on the defined academic year for this source. It must not exceed the year end date. This can be left as is if using a sync that has already been created in SDS.                                                                                                            |
 | subscriptionId                   | String   | The subscription where the ADF assets are located.                                                                                                                                                                                                                                                                           |
 
-Once the ADF and storage accounts got created Under stonerostersds create a container name it as refcodes to import refCodes.csv used for subject in classes.  (If using optionalFilesRequired parameter values as “true”) 
+Once the ADF and storage accounts are created under stonerostersds, create a container named as refcodes to import refCodes.csv used for subject in classes.
```
Comment (AyronJohnson) [inline CustomSolutions/SDS_ADF_ETL_Integrations/OR/OneRoster_README.md:135]: Remove "as" after the word "named"

**Review by AyronJohnson — CHANGES_REQUESTED** [review]

**Review by AyronJohnson — APPROVED** [review]

**Pull request merged**